src/Components/Home.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable react/prop-types */
+import React, { useState } from 'react';
+import Team from './Team';
+import './Styles/home.css';
+
+function Home({ leagues }) {
+  const leagueList = leagues;
+  const [teamsList, setTeamsList] = useState(leagueList.filter((item) => item.id === '423e4faa-4c4b-4cf2-b1e9-03e8e5cace1f')[0].teams);
+
+  const chooseLeague = (id) => {
+    setTeamsList(() => leagueList.filter((item) => item.id === id)[0].teams);
+  };
+
+  return (
+    <section>
+      <div className="navigator">
+        {leagueList.map((item) => (
+          <button key={item.id} type="button" onClick={() => { chooseLeague(item.id); }}>
+            <img src={item.logo} style={{ width: '50px', height: '60px' }} alt="League logo" />
+            <h5>{item.name}</h5>
+            <p>{item.country}</p>
+          </button>
+        ))}
+      </div>
+      <p className="identifier">TEAMS</p>
+      <div className="teams">
+        {teamsList.map((team) => (
+          <Team key={team.id} team={team} />
+        ))}
+      </div>
+    </section>
```

Comment:
- [ ]  [OPTIONAL] it would be better if you stick to the design and make two cards in a row as in the design 
- Yours 
<img width="395" alt="Screen Shot 2022-10-14 at 20 07 25" src="https://user-images.githubusercontent.com/61790587/195922738-96192400-f7ee-499f-9390-dc2ad61749f3.png">
- Design 
<img width="208" alt="Screen Shot 2022-10-14 at 20 09 30" src="https://user-images.githubusercontent.com/61790587/195922901-4478f61c-9ef9-4cfb-a008-c04badf7052a.png">